src/main/webapp/js/finscholar/commonlistview/commonlistview.soy

```diff
@@ -33,22 +33,28 @@
         jsaction="commonlistview.clickAction;
         dblclick:commonlistview.doubleClickAction">
       <span class="mdl-list__item-primary-content">
-        <i class="material-icons mdl-list__item-avatar" id="avatar">star</i>
+        <i class="material-icons mdl-list__item-avatar" id="avatar">school</i>
         <span class="cell-name">{$propsofoneitem[1]}</span>
         <span class="mdl-list__item-text-body">
-          {$propsofoneitem[3]}
+          <i class="material-icons">place</i>{$propsofoneitem[3]} <br/>
+          {$propsofoneitem[4]}
         </span>
       </span>
-      <span class="mdl-list__item-secondary-content side-prop">{$propsofoneitem[2]}</span>
+      <span class="mdl-list__item-secondary-content">
+        {$propsofoneitem[2]}
+      </span>
     </li>
   {/for}
 {/template}
 
 {template .loading}
-  <div class="mdl-spinner mdl-js-spinner is-active"></div>
-  <p> Loading more items ... </p>
+  <div id="loading">
+    <div id="p2" class="mdl-progress mdl-js-progress mdl-progress__indeterminate"></div>
+  </div>
 {/template}
 
 {template .endoflist}
-  <p> This is the end of the list. </p>
+  <div id="list-inner-footer">
+    <p> This is the end of the list. </p>
```

Comment:
nit: can you change the wording to something more intuitive like "Displaying all items." Optionally, I don't think you need a footer at all.

Reply:
Sure fixed.

Reply:
not seeing any change.